src/main/java/org/truffleruby/core/kernel/KernelNodes.java

```diff
@@ -1768,6 +1768,10 @@ public String toHexString(DynamicObject value) {
     @CoreMethod(names = {"to_s", "inspect"}) // Basic inspect, refined later in core
     public abstract static class ToSNode extends CoreMethodArrayArgumentsNode {
 
+        public static ToSNode create() {
+            return KernelNodesFactory.ToSNodeFactory.create(null);
+        }
```

Comment:
Could you update other places creating a ToSNode with `ToSNodeFactory.create(null)` to use this new method?

Reply:
Yes, updated the other usages.